firefox-ios/Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -238,11 +238,19 @@ class SearchViewController: SiteTableViewController,
         profile.firefoxSuggest?.interruptReader()
 
         let tempSearchQuery = searchQuery
+        var providers = [SuggestionProvider]()
+        let availableSuggestionTypes = FxNimbus.shared.features.firefoxSuggestFeature.value().availableSuggestionsTypes
```

Comment:
@linabutler  i know we talked about using `FeatureHolder` as a field of the class, but it doesn't seem that I need it for testing, so i opted not to.

Reply:
We generally don't prefer to directly access FxNimbus.shared in the app, so we create an interface layer type thing. Especially if this is going to hold more values than just this. Is this going to hold more values than just this?

Reply:
ah okie doke, and yes, i think eventually there will be more suggestion type/providers

Reply:
Yeah, so then let's create a layer for that. `NimbusOnboardingFeatureLayer` could be an example. :)